viewer/src/components/Game.vue

```diff
@@ -132,7 +132,7 @@ import { parseCommands } from "../logic/recent";
     ScoringBoard,
     SpaceMap,
     TurnOrder,
-    Charts,
+    Charts: () => import('./Charts.vue'),
```

Comment:
You also need to remove the `import Charts from "./Charts.vue";` at the top of the scripts for the dynamic load to be meaningul